test/test.js

```diff
@@ -171,6 +171,18 @@
         });
         checkApi('sameType', ['not']);
 
+        describe('is.set', function() {
+            it('should return true if passed parameter type is set', function() {
+                var set = new Set(['test']);
```

Comment:
You'd want to construct the test so that it doesn't does throw an error in older enviros.